dlgr/griduniverse/game_config.yml

```diff
@@ -1,6 +1,18 @@
 ---
 player_config:
+  # The distribution model used when spawning players on the grid.
+  # Griduniverse provides a pre-defined set of distribution options:
+  #    - random
+  #    - sinusoidal
+  #    - horizontal
+  #    - vertical
+  #    - edge_bias
+  #    - center_bias
+  #
+  # See the distributions.py module for their implementations
   probability_distribution: "random"
```

Comment:
Maybe we should document how to make a new distribution method? I know Natalia was interested in creating some sort of clustering distribution where items are more likely to appear near other items of the same type.